### PR TITLE
issue133_support_for_intime : Updating initial support

### LINKED ISSUE #133: support_for_intime
rtiperftest must support the compilation and execution of perftest for intime platforms.

### PR DESCRIPTION
**Pull request by sarath-rti**
This pull request is coming from the issue #133. When the pull request is merged into "master" the issue will be close #133

### CODE REVIEW

**Comment by sarath-rti** [comment]
Please review

File: srcCpp/perftest_publisher.cxx [hunk 1]
```diff
@@ -2212,6 +2212,23 @@ inline void perftest_cpp::SetTimeout(unsigned int executionTimeInSeconds,
       #ifdef RTI_WIN32
         CreateTimerQueueTimer(&_hTimer, _hTimerQueue, (WAITORTIMERCALLBACK)Timeout_scan,
                 NULL , executionTimeInSeconds * 1000, 0, 0);
+      #elif defined RTI_INTIME
+            HANDLE  hTimer;
+            hTimer = RtCreateTimer(NULL, 0, Timeout_scan, NULL,
```
Comment (softwarejimenez) [inline srcCpp/perftest_publisher.cxx]: hTimer = RtCreateTimer(NULL,
        0, 
        Timeout_scan,
        NULL, 
        RT_PRIORITY_MAX, CLOCK_FASTEST);

File: srcCpp/perftest_publisher.cxx [hunk 2]
```diff
@@ -2212,6 +2212,23 @@ inline void perftest_cpp::SetTimeout(unsigned int executionTimeInSeconds,
...
+                        RT_PRIORITY_MAX, CLOCK_FASTEST);
+            if (NULL == hTimer) {
+                    printf("Can not create the timer!!\n");
+                    exit(1);
+            }
+            LARGE_INTEGER timeout;
+            // set timer absolute
+            if (!RtGetClockTime(0, &timeout))
+                printf("Cannot get clock time!!\n");
+            timeout.QuadPart += executionTimeInSeconds*1000*10000;   /*Time in 100 nano seconds */
```
Comment (softwarejimenez) [inline srcCpp/perftest_publisher.cxx]: timeout.QuadPart += executionTimeInSeconds * 1000 * 10000;   /*Time in 100 nano seconds */

File: srcCpp/perftest_publisher.cxx [hunk 3]
```diff
@@ -2212,6 +2212,23 @@ inline void perftest_cpp::SetTimeout(unsigned int executionTimeInSeconds,
...
+            if (!RtSetTimer( hTimer, &timeout, NULL)) {
```
Comment (softwarejimenez) [inline srcCpp/perftest_publisher.cxx]: if (!RtSetTimer(hTimer, &timeout, NULL)) {

File: srcCpp/perftest_publisher.cxx (hunk 2 above)
Comment (sarath-rti) [reply srcCpp/perftest_publisher.cxx]: I have checked in new changes, but i dont know how to apply those changes to this pull request.
Please can you update the pull request with new changes?

**Comment by javmorcas** [comment]
This branch is not up to date, therefore I am taking the changes and merging them into the latest state of master. See Pull Request here #192.

**Pull request closed without merging**